docs/applications/development/how-to-unbundle-nginx-from-omnibus-gitlab-for-serving-multiple-websites.md

```diff
@@ -0,0 +1,195 @@
+
+1.  To unbundle nginx from GitLab, we'll need to disable the version included in the Omnibus package. Add the following lines to `/etc/gitlab/gitlab.rb`:
+
+    {: .file-excerpt}
+    /etc/gitlab/gitlab.rb
+    :   ~~~
+	    # Unbundle nginx from Omnibus GitLab
+	    nginx['enable'] = false
+	    # Set your Nginx's username
+	    web_server['external_users'] = ['www-data']
+        ~~~
+
+2.  Reconfigure GitLab to apply the changes:
+
+	    sudo gitlab-ctl reconfigure
+
+For more information on how to customize Omnibus Nginx please [click here](https://gitlab.com/gitlab-org/omnibus-gitlab/blob/master/doc/settings/nginx.md) to read the official documentation.
+
+## Install Ruby, Passenger, and Nginx
+
+Well done! At this point, GitLab's bundled nginx has been disabled. The next step is to install and configure the web server from scratch.
+
+1.  Since GitLab is written in Ruby, install Ruby on your system:
+
+		sudo apt-get install ruby
+		sudo gem install rubygems-update
+		sudo update_rubygems
+
+2.  We'll also need to install Phusion Passenger, a web application server for Ruby. Install Phusion Passenger's PGP key:
+		
+        sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 561F9B9CAC40B2F7
+		
+3.	Add Passenger's APT repository by adding the following lines to `/etc/apt/sources.list.d/passenger.list`:
+		
+    {: .file}
+    /etc/apt/sources.list.d/passenger.list
+    :   ~~~
+        deb https://oss-binaries.phusionpassenger.com/apt/passenger trusty main
+	    ~~~
+
+    {: .note}
+    > If you're using Ubuntu 16.04, replace `trusty` with `xenial` in the above command.
+        
+4.  Update your package repositories:
+
+    	sudo apt-get update
+
+5.	Install Passenger and nginx:
+		
+        sudo apt-get install nginx-extras passenger
+
+6.  Enable your brand new Passenger module by uncommenting the `include /etc/nginx/passenger.conf;` line from the `/etc/nginx/nginx.conf` file:
+
+    {: .file-excerpt}
+    /etc/nginx/nginx.conf
+    :   ~~~ conf
+        include /etc/nginx/passenger.conf;
+        ~~~
+
+4.  Finally, restart nginx. On Ubuntu 14.04:
+
+		sudo service nginx restart
+
+    On Ubuntu 16.04:
+
+        sudo systemctl restart nginx
+
+For further information, please refer to [Installing Passenger + Nginx on Ubuntu 14.04 LTS (with APT)](https://www.phusionpassenger.com/library/install/nginx/install/oss/trusty/ "Installing Passenger + Nginx").
+
+## Create a New Virtual Host
+
+In this section, we'll create a new virtual host to serve GitLab. Since we unbundled nginx, we'll also be able to configure other virtual hosts for other websites and apps.
+
+1.  Copy the default virtual host file to a new virtual host file, replacing `example.com` with the name you'd like to use for your virtual host:
+
+	    sudo cp /etc/nginx/sites-available/default /etc/nginx/sites-available/example.com
+
+2.  Edit your new virtual host file to match the following, replacing `example.com` with your own hostname:
```

Comment:
If the file excerpt below has the only directives we need, why do we copy the default host first?